lib/components/viewers/route-viewer.js

```diff
@@ -159,11 +164,13 @@ class RouteRow extends PureComponent {
           onClick={this._onClick}
         >
           <div style={{display: 'inline-block'}}>
-            {// TODO: re-implement multi-agency logos for route viewer.
-              // Currently, the agency object is not nested within the get all
-              // routes endpoint and causing this to only display transitOperators for
-              // the selected route.
-              // operator && <img src={operator.logo} style={{marginRight: '5px'}} height={25} />
+            {operator &&
```

Comment:
On the configs that don't define a logo, this results in a "broken image link" placeholder to be shown, so the condition for showing `<img>` should rather be `operator && operator.logo`.

Reply:
If no logo is found, should some other indicator for agency be used (e.g., name)?